cmd/tailscale/cli/web.go

```diff
@@ -91,6 +91,10 @@ func runWeb(ctx context.Context, args []string) error {
 	hasPreviewCap := st.Self.HasCap(tailcfg.CapabilityPreviewWebClient)
 
 	cliServerMode := web.LegacyServerMode
+	var wantWebClient bool
+	if prefs, err := localClient.GetPrefs(ctx); err == nil {
```

Comment:
Actually I guess we could just not even set the pref at all if it was already set?

Reply:
done